README.md

```diff
@@ -1,11 +1,17 @@
-ament_lint contains a number of packages that perform linting tasks. Those linters can be used both from the command line and from CMake.
+## ament_lint
+
+ament_lint contains a number of packages, each package wraps a linter into an ament_$linter CLI tool, which is then wrapped in CMake (ament_cmake_$linter)
+Those linters can be used both from the command line and from CMake.
```

Comment:
They also have a Python API, e.g., here: https://github.com/ament/ament_lint/blob/2bee9151aba25b94020241c00f468f12944ab265/ament_mypy/ament_mypy/main.py#L30. This can be called in a test, e.g., here: https://github.com/ros2/launch/blob/f8ef67a0f8cface863e1cd168f0b6bf2976be60b/launch/test/test_mypy.py#L22.